code/modules/admin/secrets.dm

```diff
@@ -43,6 +43,8 @@
 
 	if(check_rights(R_FUN,0))
 		dat += {"
+			<B>Fun Secrets</B><BR>
+			<BR>
```

Comment:
Readd the AK47s for everyone button somewhere around here, and I'll call you a good boy~